### PR TITLE
82 add fetchemail action and send email from the logged user

### LINKED ISSUE #82: Add FetchEmail action
**Short description:**
Add the action EmailFetched from the service Gmail .

**Describe the solution you'd like**
When an email is fetched, possibility to raise an event

### PR DESCRIPTION
**Pull request by GuyomT**
# Purpose

The purpose of this pull request is to send an email from the logged gmail user

Fixes #82 

## Added features

possibility to send an email given the access token of the user after he has accepted the app to send emails on his behalf
incomming : add the action of get an email 

## Updated features

add scopes send email in the gmail api login

### CODE REVIEW

**Review by LucasTesnier — DISMISSED** [review]
!! DOESN'T COUNT FOR A REVIEW CAUSE I'M ALSO A CONTRIBUTOR !!

**Review by ZQUEMA — DISMISSED** [review]

File: server/services/gmail/actions/fetch_email.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const gmail = require('../gmail_init').getGmailClient();
+// Imports the Google Cloud client library
```
Comment (tbellicha) [inline server/services/gmail/actions/fetch_email.js]: Useless comment

File: server/services/gmail/actions/fetch_email.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const request = {
+    'labelIds': ['INBOX'],
+    'topicName': 'projects/a-equal-l-squared/topics/watch-emails-sub'
+  }
+  const res =
+      gmail.users.watch({auth: gmail.auth, userId: userId, resource: request})
+  console.log(`New email received: ${JSON.stringify(res)}`);
+}
+
+async function watchNewEmails() {
```
Comment (tbellicha) [inline server/services/gmail/actions/fetch_email.js:22]: Missing doc

File: server/services/gmail/actions/fetch_email.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    });
+
+    console.log(`Watch resource: ${res.data.resourceId}`);
+    console.log(`Expiration: ${res.data.expiration}`);
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+async function getNewEmail() {
```
Comment (tbellicha) [inline server/services/gmail/actions/fetch_email.js:43]: Missing doc

**Review by tbellicha — COMMENTED** [review]
There is a useless comment and 2 functions missing doc

**Review by tbellicha — APPROVED** [review]

**Review by TomDUVAL-MAHE — APPROVED** [review]

**Pull request merged**